tests/Extensions.Hosting.AsyncInitialization.Tests/AsyncInitializationTests.cs

```diff
@@ -12,35 +20,35 @@ public class AsyncInitializationTests
         [Fact]
         public async Task Single_initializer_is_called()
         {
-            var initializer = A.Fake<IAsyncInitializer>();
+            IAsyncInitializer initializer = A.Fake<IAsyncInitializer>();
```

Comment:
Please revert to `var`. I won't mention future occurrences of this, but please revert them as well.